unified-runtime/test/conformance/exp_command_buffer/enqueue.cpp

```diff
@@ -42,6 +42,22 @@ struct urEnqueueCommandBufferExpTest
     ASSERT_SUCCESS(urEnqueueUSMFill(queue, device_ptr, sizeof(zero_pattern),
                                     &zero_pattern, allocation_size, 0, nullptr,
                                     nullptr));
+
+    const int num_copy_buffers = 10;
+    const int buffer_size = 512;
+    int* dst_buffers[num_copy_buffers];
+    int* src_buffers[num_copy_buffers];
+
+    std::vector<int> temp_val(buffer_size, 3);
+
+    for (int i = 0; i < num_copy_buffers; i++) {
+      ASSERT_SUCCESS(urUSMDeviceAlloc(context, device, nullptr, nullptr, buffer_size * sizeof(int), (void**) &(dst_buffers[i])));
+      ASSERT_SUCCESS(urUSMDeviceAlloc(context, device, nullptr, nullptr, buffer_size * sizeof(int), (void**) &(src_buffers[i])));
```

Comment:
These allocations are being leaked. The `dst_buffers`/`src_buffers` arrays should be a class members, and then the elements feed in `TearDown` with `urUSMFree`